pkg/http/server.go

```diff
@@ -96,6 +99,21 @@ func (s *server) useCors(next fasthttp.RequestHandler) fasthttp.RequestHandler {
 	return corsHandler.CorsMiddleware(next)
 }
 
+func useAPIAuthentication(next fasthttp.RequestHandler) fasthttp.RequestHandler {
+	token := auth.GetAPIToken()
+	if token != "" {
```

Comment:
Can we do `return next` earlier? Let's try to reduce the depth. the latest version of go linter will complain it.

Reply:
I'm not sure what you mean

Reply:
e.g.

```
if token == "" {
   return next
}

return func(ctx *fasthttp.RequestCtx) {
    v := ctx.Request.Header.Peek(auth.APITokenHeader)
..
```

Reply:
What won't we do for the linter..

Reply:
"fixed"